docs/modules/ROOT/pages/how-to-guides/Troubleshooting.adoc

```diff
@@ -0,0 +1,120 @@
+= Troubleshooting Application Deployment Issues on Payara Cloud
+
+Deploying a new application to Payara Cloud may occationally be accompanied by errors or issues that prevent the application from deploying and running smoothly. Below is a comprehensive guide to help you troubleshoot some of these common problems.
+
+[[no-deployed-application-artefact]]
+== No Deployed Application Artifact
+
+=== Symptoms
+You've initiated the deployment process, but there is no sign of your application artifact in the Payara Cloud Dashboard.
+
+=== Troubleshooting Steps
+1. **Check Upload Status**: Make sure the artifact has been successfully uploaded. Look for any error messages during the upload process.
+2. **Inspect Logs**: Check the deployment logs for any error messages that could indicate what went wrong.
+3. **Verify Artifact Type**: Ensure that you are uploading a supported artifact type (.war) created from a supported Jakarta EE API version (10, 8).
+
+[[application-not-reachable]]
+== DNS Propagation for Application URL Not Fully Done
+
+=== Symptoms
+Your application has been deployed, but navigating to the provided URL results in a DNS error.
```

Comment:
Last step should be "If DNS is propagated, it can still be your router or computer. Flush DNS resolver cache with  e. g. `ipconfig /flushdns` in windows."